python/lsst/dax/imgserv/getimage/imagegetter_v1.py

```diff
@@ -462,56 +463,34 @@ def _apply_cutout(self, src_img, metadata, ra, dec, width, height, data_id,
         pix_h = height * pixel_per_arcsec
         self._log.debug("ra=%f dec=%f xy_wcs=(%f,%f) xyCenter=(%f,%f)",
                 ra, dec, xy_wcs.getX(), xy_wcs.getY(), xy_center_x, xy_center_y)
-        cutout = self._cutout_from_src(src_img, xy_center_x, xy_center_y,
+        cutout = self._cutout_from_src(data_id, src_img, xy_center_x, xy_center_y,
                 pix_w, pix_h, wcs)
         return cutout
 
     def _metadata_from_data_id(self, data_id):
         # Return the metadata for the query results in qResults and a butler.
-        if self._butler_keys == sorted(["run", "camcol", "field", "filter"]):
-            metadata = self._butler.get(self._imagedataset_md(),
-                    run=data_id["run"],
-                    camcol=data_id["camcol"],
-                    field=data_id["field"],
-                    filter=data_id["filter"])
-        elif self._butler_keys == sorted(["tract", "patch", "filter"]):
-            metadata = self._butler.get(self._imagedataset_md(),
-                    tract=data_id["tract"],
-                    patch=data_id["patch"],
-                    filter=data_id["filter"])
-        else:
-            # no metadata found for the specified data id
-            metadata = None
+        metadata = self._butler.get(self._imagedataset_md(),dataId=data_id)
```

Comment:
Space after comma.

It looks like this could fail if the `data_id` is unrecognized, whereas before it would return `None`.  Is this a problem?